src/AnnouncementBubble/NormalAnnouncementBubble.less

```diff
@@ -149,11 +156,39 @@ button.NormalAnnouncementBubble--deleteMenuItem {
   white-space: nowrap;
 
   .tooltip-inner {
-    max-width: 30rem;
+    width: 18.75rem;
```

Comment:
What was this change for?

Reply:
So this is one thing I had a question on. I know for M1 Nick said that we should Tooltip width to be 300px=18.75rem but for some reason the width would not change when I tried to change the component. Honestly I think the tooltip width looks fine as it is, but was wondering what your thoughts were.

Reply:
I dont quite understand this part - "the width would not change when I tried to change the component"